fake_spectra/parallel.py

```diff
@@ -0,0 +1,47 @@
+""" 
+- An example on how to use MPI capability of the fake_spectra
+- This example has been used on TNG300-1 to get many spectra with and without removing DLAs in a very short time
+- Current version of the MPI feature only supports the Illustris simulations with hdf5 outputs
+"""
+import fake_spectra
+from fake_spectra.randspectra import RandSpectra
+from fake_spectra.ratenetworkspectra import RateNetworkGas
+from mpi4py import MPI
+import numpy as np
+import time
+import argparse
+
+def get_spec(num, base, savefile, savedir, res, numlos, thresh):
```

Comment:
Docstring

Reply:
It is much easier now to run the MPI feature, I added instructions on `README.md`. So I deleted this file.